teammapper-frontend/src/app/core/services/mmp/mmp.service.ts

```diff
@@ -471,12 +471,20 @@ export class MmpService implements OnDestroy {
       case 'png': {
         const image = await this.exportAsImage(format);
 
-        UtilsService.downloadFile(
-          `${name}.${format === 'jpeg' ? 'jpg' : format}`,
-          image
-        );
+        if (image === 'data:,') {
```

Comment:
cool thanks. Refactoring suggestions:

1) Make a constant EMPTY_IMAGE_DATA where you put the "data:..."
2) Extract method refactoring would be nice here to make this already big method smaller.